bentoml/saved_bundle/bundler.py

```diff
@@ -142,6 +142,13 @@ def save_to_dir(bento_service, path, version=None, silent=False):
             )
         )
 
+    # copy custom webui if enabled
```

Comment:
My instinct is to add a property to the `bento_service`, e.g.

```
@property
def static_files(self):
    """Returns static files if present.

    Raises:
        AttributeError if no static files
    """
    return self._static_files
```

but the maintainers can advise you more.

Motivation: underscore-prefixed attributes are "private", so it doesn't make sense to access them externally like this.

Reply:
Done!